openapi_resource_definitions/top_level_list_resources/TicketList/get.yaml

```diff
@@ -2,6 +2,59 @@ tags:
 - Ticket
 summary: Get the list of Tickets
 description: Return the tickets table data.
+parameters:
+  - name: aportio_id
+    in: query
+    description: Search for a ticket with this Aportio ID.
+    required: false
+    schema:
+      type: integer
+    example: "?aportio_id=12345"
+  - name: customer_id
+    in: query
+    description: Search for tickets that are associated with this customer ID.
+    required: false
+    schema:
+      type: integer
+    example: "?customer_id=3"
+  - name: user_id
+    in: query
+    description: Search for tickets that are associated with this user ID.
+    required: false
+    schema:
+      type: integer
+    example: "?user_id=3"
+  - name: short_title
+    in: query
+    description: Search for tickets that have this title.
+    required: false
+    schema:
+      type: string
+    example: "?short_title=3"
+  - name: status
+    in: query
+    description: Search for tickets that have this status.
+    required: false
+    schema:
+      type: string
+      enum:
+        - OPEN
```

Comment:
I noticed we have quotes in other instances of `enum`, but not here?

Reply:
I must have forgot in my hurry, however this okay with regards to yaml syntax because these are treated as a string anyway, in fact the quotes are removed in Swaggerhub. I've add the quotes into the other `enum` areas to be consistent